code/game/objects/structures/displaycase.dm

```diff
@@ -107,7 +109,7 @@
 
 /obj/structure/displaycase/proc/trigger_alarm()
 	set waitfor = FALSE
-	if(alert && is_station_contact(z))
+	if(alert && (is_station_contact(z) || force_alarm))
```

Comment:
Why does this even have a level check?

Reply:
So AI's are not notified when the alarm goes off if their is an alarmed display case on another zlvl. Which naturally, blocks the alarm from working at all. Very silly, probably could be removed.